src/datastore/pg-store.ts

```diff
@@ -1910,8 +1910,12 @@ export class PgStore extends BasePgStore {
     poxTable: PoxSyntheticEventTable;
   }): Promise<DbPoxSyntheticEvent[]> {
     return await this.sqlTransaction(async sql => {
+      const cols =
+        poxTable === 'pox4_events'
+          ? [...POX_SYNTHETIC_EVENT_COLUMNS, 'signer_key']
```

Comment:
Probably not needed, but this could be an array.push, depending on how often this code path runs, to avoid a spread